backend/init_data/skills/pptx-slide/create_pptx.py

```
@@ -0,0 +1,826 @@
+                "2) Reducing the number of slides, "
+                "3) Using clearer bullet point formatting."
+            ),
+        }
+
+    def _generate_pptx_code(
+        self,
+        content: str,
+        title: Optional[str],
+        template: str,
+        output_path: str,
+        previous_error: Optional[str] = None,
```

Ruff (0.14.11)

Unused method argument: `previous_error`

(ARG002)